src/runtime/cli/config/configuration-qemu.toml.in

```diff
@@ -356,6 +356,16 @@ valid_entropy_sources = @DEFVALIDENTROPYSOURCES@
 # See: https://www.qemu.org/docs/master/qemu-qmp-ref.html#Dump-guest-memory for details
 #guest_memory_dump_paging=false
 
+# Enable swap in the guest. Default false.
+# When enable_guest_swap is enabled, insert a raw file to the guest as the swap device
+# if the swappiness of a container (set by annotation "io.katacontainers.resource.swappiness")
+# is bigger than 0.
+# The size of the swap device should be 
+# swap_in_bytes (set by annotation "io.katacontainers.resource.swap_in_bytes") - memory_limit_in_bytes.
```

Comment:
Should these annotation keys be documented in `how-to-set-sandbox-config-kata.md` ?

Reply:
We just have two annotations "io.katacontainers.resource.swap_in_bytes" and "io.katacontainers.resource.swappiness" for container.
Both of them are the swap options.
Do you mind I introduce them in the doc for swap when I add the doc?

Reply:
I added a special commit that add swap annotations introduction of
    `io.katacontainers.config.hypervisor.enable_guest_swap`,
    `io.katacontainers.container.resource.swappiness"` and
    `io.katacontainers.container.resource.swap_in_bytes"` to
    how-to-set-sandbox-config-kata.md.